### PR TITLE
feat: other service method options in confirm service (CMC-356)

### PR DESCRIPTION
**Pull request by swalker125**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/CMC-356

### Change description ###

Configured document exchange, fax, email and other journies for confirm service.

BREAKING CHANGE: serviceMethod field -> complex type will break any existing cases in local / demo / aat.

**Does this PR introduce a breaking change?** (check one with "x")

```
[x] Yes
[ ] No
```

### CODE REVIEW

**Comment by swalker125** [comment]
still todo: validate serviceDate is in the past and prior to date of issue

will address the updates for working days in a separate PR if they are not already completed

File: ccd-definition/CaseField.json [hunk 1]
```diff
@@ -119,8 +119,7 @@
     "CaseTypeID": "UNSPECIFIED_CLAIMS",
     "ID": "serviceMethod",
     "Label": "How did you serve the documents?",
-    "FieldType": "FixedRadioList",
-    "FieldTypeParameter": "ServiceMethod",
+    "FieldType": "ServiceMethod",
```
Comment (AlisonRevittHM) [inline ccd-definition/CaseField.json:122]: This should be respondentSolicitor1ServiceMethod

File: ccd-definition/CaseField.json [hunk 2]
```diff
@@ -139,6 +138,14 @@
     "FieldType": "Date",
     "SecurityClassification": "Public"
   },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "UNSPECIFIED_CLAIMS",
+    "ID": "serviceDateAndTime",
```
Comment (AlisonRevittHM) [inline ccd-definition/CaseField.json:144]: This should be respondentSolicitor1ServiceDateTime

**Review by AlisonRevittHM — CHANGES_REQUESTED** [review]
data review

File: ccd-definition/CaseField.json (hunk 2 above)
Comment (swalker125) [reply ccd-definition/CaseField.json:144]: @AlisonRevittHM if you expand this file there are actually lots more instances where it's just service, i.e serviceLocation. So I am not inclined to change all of that within this PR. Could you raise a separate JIRA ticket please

File: src/main/resources/application.yaml [hunk 3]
```diff
@@ -68,6 +68,8 @@ core_case_data:
 
 spring:
   profiles: local
+  flyway:
```
Comment (dharmendrak) [inline src/main/resources/application.yaml]: please remove it

**Review by dharmendrak — DISMISSED** [review]

File: src/main/resources/application.yaml (hunk 3 above)
Comment (swalker125) [reply src/main/resources/application.yaml]: good spot, removed

**Review by dharmendrak — APPROVED** [review]

**Pull request merged**